CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ Be sure to select the *Source* option from the installation preferences.
 
 - We recommend developing on Linux as this is the only OS where all features are currently 100% functional. Avoid introducing changes that will only work on a particular OS, as we're currently working to support MacOSX and Windows. Changes to existing code that improve cross-platform compatibility are most welcome!
 
-- Use Python >= 3.6 for development, as these are the only versions where AutoGluon is fully functional.
+- Use Python >= 3.7 for development, as these are the only versions where AutoGluon is fully functional.
```

Comment:
Can be more specific: Use Python 3.7, 3.8, and 3.9 for development,

Reply:
Done.

Reply:
Unit test has failed on OOM.  Pretty sure unrelated to my code.